boost.sh

```diff
@@ -850,6 +856,38 @@ scrunchAllLibsTogetherInOneLibPerPlatform()
     done
 }
 
+buildUniversalStatic()
+{
+    if [[ -n $UNIVERSAL ]]; then
```

Comment:
I would prefer it if you made this check at line 1088 where `buildUniversalStatic()` is called, rather than always calling this function & only actually executing it if the check succeeds.